README.md

```diff
@@ -1,25 +1,28 @@
 Bitcoin XT
 ==========
 
-Bitcoin XT is an implementation of a Bitcoin full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
+Bitcoin XT is an implementation of a Bitcoin Cash (BCH) full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
 
-XT uses the same data directories as Core so you can easily switch back and forth. You will keep and continue updating the same block chain.
+Bitcoin XT downloads are code signed and are [built reproducibly using gitian](https://github.com/bitcoinxt/gitian.sigs). Discussion can be found at the [XT Gitter](https://gitter.im/bitcoinxt/Lobby).
+
+Data Directory Compatibility
+============================
+
+If using a data directory from a different BCH or BTC full node, specify `-reindex` on first launch (reason: UTXO database format).
+
+Set aside `wallet.dat` from Bitcoin Core or Bitcoin ABC (reason: HD wallet not yet supported). Also delete `fee_estimates.dat` from these clients.
  
-Bitcoin XT downloads are code signed and are [built reproducibly using gitian](https://github.com/bitcoinxt/gitian.sigs). If you use it please [sign up to the announcement mailing list](https://bitcoinxt.software) so you can be reminded of new versions.
+Bitcoin XT can run in BTC chain mode by specifying `-uahftime=0`. Please note that Bitcoin XT does not validate segwit, and the conversion notes above still apply.
+
 
 Development process
 ===================
 
-To propose a patch for inclusion or to discuss Bitcoin development in general, you are welcome to use the [mailing list](https://groups.google.com/forum/#!forum/bitcoin-xt/).
+Ideas for useful features are tracked in Issues.  Pull requests from developers, including well-done cherry picks from other clients or cryptos, are very welcome.
+
+To discuss Bitcoin Cash development in general, you are welcome to use the [mailing list](https://lists.linuxfoundation.org/pipermail/bitcoin-ml/).
```

Comment:
Remove mailing list

Reply:
This is the new cross-implementation mailing list link.

Reply:
My bad :-)